pkg/kubectl/cmd/get.go

```diff
@@ -182,13 +183,22 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 		return cmdutil.UsageError(cmd, usageString)
 	}
 
-	// always show resources when getting by name or filename
+	// determine if args contains "all"
+	for _, a := range args {
+		if a == "all" {
```

Comment:
@fabianofranz 
> Remember you may be interesting in getting all + something that is not in all, in which case you would have for example kubectl get events,all. Seems to break the proposed logic.

This is a good point, I am updating the logic for `MultipleTypesRequested` in [this PR](https://github.com/kubernetes/kubernetes/pull/40046/files), and while specifying something like `$ kubectl get pods,all` does not fulfill the base case check of `if len(args) == 1 && args[0] == "all"`, the rest of the logic for that function does correctly detect multiple types in cases like this one when it splits the string by `,` and finds more than one "resource" was specified. Although this would no longer be work for this PR (if we had just decided to check for "all" using `len(args) == 1 && args[0] == "all"` rather than calling the entire `MultipleTypesRequested` helper)